component/openshiftscc.jsonnet

```diff
@@ -0,0 +1,53 @@
+// instance-specific security context constraint object for openshift
+local kap = import 'lib/kapitan.libjsonnet';
+local kube = import 'lib/kube.libjsonnet';
+local inv = kap.inventory();
+// The hiera parameters for the component
+local params = inv.parameters.nfs_subdir_external_provisioner;
+
+local isOpenshift = std.startsWith(inv.parameters.facts.distribution, 'openshift');
+local instance = inv.parameters._instance;
+
+local nfsMountScc = {
+  apiVersion: 'security.openshift.io/v1',
+  kind: 'SecurityContextConstraints',
+  metadata: {
+    name: 'hostmount-anyuid-' + instance,
+    namespace: params.namespace,
```

Comment:
I'd suggest adding the recommended labels according to https://syn.tools/syn/references/labels-guide.html so it's explicitly clear that the SCC is Syn-managed and which component is responsible for it.

Reply:
I added:
```
'app.kubernetes.io/name': 'hostmount-anyuid-' + instance,                                                                                                           
'app.kubernetes.io/managed-by': 'syn',                                                                                                                              
```
Should I also add sth like:
```
app.kubernetes.io/part-of: syn-component-nfs-subdir-external-provisioner
```
Also I'm unsure what 'app.kubernetes.io/instance' should refer to in this context.

Reply:
I think just `app.kubernetes.io/managed-by: syn` and `app.kubernetes.io/part-of: syn-component-nfs-subdir-external-provisioner` would be sufficient.

Alternatively, you could have `app.kubernetes.io/instance` refer to the nfs-subdir-external-provisioner instance while `app.kubernetes.io/component` could be set to `nfs-subdir-external-provisioner`. In that case, you could omit the `part-of` label.

Reply:
Thanks, I went with the "alternative" approach to make it as clear as possible.

Reply:
See my suggestion, I would just do `'app.kubernetes.io/instance': instance`